controls/V-72241.rb

```diff
@@ -59,7 +59,14 @@
 The SSH service must be restarted for changes to take effect."
   tag "fix_id": "F-78595r3_fix"
 
-  describe sshd_config do
-    its('ClientAliveCountMax') { should cmp '0' }
+  if os.release.to_f >= 7.4
+    impact 0.0
+    describe "The release is #{os.release}" do
+      skip "The release is newer than 7.4; this control is Not Applicable."
```

Comment:
You need to update the impact logic as well so that this buckets to Not Applicable when NA.

Reply:
Impact is set to `0.0` the line above.